.agents/hooks/run_dylint_after_rust_edit.py

```
@@ -1,49 +1,78 @@
 
+import argparse
 import fcntl
 import json
 import os
 import re
 import subprocess
 import sys
 import time
+from dataclasses import dataclass
 from pathlib import Path
+from typing import Callable
```

Ruff (0.15.11)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)